lang/kotlin/Portfile

```diff
@@ -19,10 +19,10 @@ long_description    Kotlin is a pragmatic programming language for JVM \
                     and is focused on interoperability, safety, clarity \
                     and tooling support.
 
-homepage            http://kotlinlang.org/
+homepage            https://kotlinlang.org/
```

Comment:
This is *very* good. We should probably check for https versions of all our URLs.